### PR TITLE
New package: q5go-2.12

### PR DESCRIPTION
**Pull request by jason1987d**
#### Testing the changes
- I tested the changes in this PR: **YES**|**briefly**|**NO**

### CODE REVIEW

File: srcpkgs/q5go/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'q5go'
+pkgname=q5go
+version=2.1.2
+revision=1
+archs="i686 x86_64"
```
Comment (Chocimier) [inline srcpkgs/q5go/template:5]: Nope, figure out cross build. dropping `archs=` is first step.

Comment (Chocimier) [reply srcpkgs/q5go/template]: It's getting better :) You can check locally with `./xbps-src -a aarch64 pkg q5go`

Comment (jason1987d) [reply srcpkgs/q5go/template]: The package itself shows no documentation on which archs etc... what's the best next step as to figuring out how to get this to run on other archs? That local check is useful too.

Comment (Chocimier) [reply srcpkgs/q5go/template]: Usually one adds necessary deps/flags/whatever until it compiles. At that point we assume it runs as well if build system is common. qmake is.
If you hit something you can't figure out, push that and ask.

```
$ xlocate bin/uic
qt5-host-tools-5.15.3+20211001_2        /usr/lib/qt5/bin/uic
```

Comment (jason1987d) [reply srcpkgs/q5go/template]: Testing locally using aarch64. Change working so far is I copied the depends to hostmakedepends and now it's compiling. Though I will likely include that qt5-host-tools as a hostmakedepends as well.

**Comment by jason1987d** [comment]
I've set pull default to rebase so I won't have the issues with multiple commits. I will close this, create new as now mine builds on the other archs.

**Comment by classabbyamp** [comment]
in the future, no need to close/open PRs, just use `git push --force`

**Pull request closed without merging**